apps/web/package.json

```diff
@@ -7,14 +7,16 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint --fix './'",
-    "test": "jest",
+    "playwright": "playwright test",
+    "show": " playwright show-report",
```

Comment:
1. I think `test` should invoke both Jest and Playwright, no?
2. `show` is not descriptive enough.

Reply:
@kilemensi So for both playwright and jest invoking them will result in an error 👇🏽
<img width="1030" alt="Screenshot 2022-03-22 at 14 19 55" src="https://user-images.githubusercontent.com/3693563/159470612-c2445434-36b2-46aa-be55-38b17b434a23.png">

Reply:
@kilemensi We can look at this https://www.npmjs.com/package/jest-playwright-preset as a solution?

Reply:
> We can look at this https://www.npmjs.com/package/jest-playwright-preset as a solution?

They also recommend using Playwright test runner there. We can look into this issue once you’ve addressed all other comments @nerdlet